packages/grid/backend/backend.dockerfile

```diff
@@ -55,7 +55,7 @@ COPY --chown=$USER_GRP \
 
 # Install all dependencies together here to avoid any version conflicts across pkgs
 RUN --mount=type=cache,id=pip-$UID,target=$HOME/.cache/pip,uid=$UID,gid=$UID,sharing=locked \
-    pip install --user --default-timeout=300 torch==2.2.1 -f https://download.pytorch.org/whl/cpu/torch_stable.html && \
+    pip install --user --default-timeout=300 torch==2.2.2 && \
```

Comment:
this will install the nvidia cuda libs as well right?

Reply:
Its weird, I always thought the `cpu` extra worked but its telling me that it doesnt exist, maybe thats just for the `macos` triplet?